### ISSUE TITLE
“Thank you for trying out Minetest”—Alert bells ring instantly

### ISSUE THREAD

**Issue by Wuzzy2** [minetest/minetest.github.io#129]
“Thank you for trying out Minetest”

from <https://www.minetest.net/downloads/>.

When I read a sentence like this somewhere else, my alert bells ring instantly. Wording like “trying out” are usually used as euphemisms for demo versions. I'm worried this sentence alone may send similar-minded players running away screaming.

I suggest to reword it to something like:

“Thank you for downloading Minetest”

**Comment by paramat** [comment]
I agree, it reads awkwardly and understates MT.

**Comment by paramat** [comment]
PR #142
